plugins/jdg70/src/main/resources/server.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:4.0">
```

Comment:
Is there any difference to the previous config? I am thinking about symlinks rather than copy-paste.